Content.Server/Weapon/Melee/EnergySword/EnergySwordSystem.cs

```diff
@@ -74,6 +77,12 @@ private void TurnOff(EnergySwordComponent comp)
                 malus.Malus -= comp.litDisarmMalus;
             }
 
+            if(TryComp<MeleeWeaponComponent>(comp.Owner, out var weaponComp))
+                weaponComp.HitSound = new SoundPathSpecifier("/Audio/Weapons/genhit1.ogg");
+
+            if (TryComp<SharpComponent>(comp.Owner, out var sharpComp))
+                RemComp<SharpComponent>(comp.Owner);
```

Comment:
```suggestion
            RemComp<SharpComponent>(comp.Owner);
```
remcomp already checks this

Reply:
Nice, very good to know!